snuba/writer.py

```diff
@@ -33,7 +36,7 @@ def __row_to_column_list(self, columns, row):
             values.append(value)
         return values
 
-    def write(self, rows):
+    def write(self, rows: TableRow):
```

Comment:
```suggestion
    def write(self, rows: Iterable[TableRow]):
```

Reply:
mh, I wonder why mypy did not yell. Probably because it is spitting so many errors from other areas of Snuba that I did not notice this

Reply:
Yeah, I've been trying to think of a way to get this integrated into CI that allows us to not have to solve all of the already existing issues at once but haven't come up with any ideas yet that don't require us to build tooling around them.